samples/matter/light_bulb/CMakeLists.txt

```diff
@@ -6,6 +6,16 @@
 
 cmake_minimum_required(VERSION 3.13.1)
 
+option(BUILD_WITH_DFU "Build target with Device Firmware Upgrade support" OFF)
+if(BUILD_WITH_DFU)
+    set(OVERLAY_CONFIG ${CMAKE_CURRENT_SOURCE_DIR}/../common/config/overlay-dfu_support.conf)
```

Comment:
Use `list(INSERT ...` to allow users to provide their own overlays on top of ours.